### PR TITLE
Fixes #2033 - Added mode view

### LINKED ISSUE #2033: [triage dashboard][feedback] Cards vs lists
Not a bug, just a personal perference.

Right now the design is using cards that require quite a bit of scrolling to interact with. I would prefer a flat list. The most important info to me is:

`Bug Number | <a href='link to bug'>Bug title`

Dunno if it's worth adding view options, or we should move to a simpler design?

Thoughts @karlcow @magsout?

### PR DESCRIPTION
**Pull request by magsout**
r @miketaylr and @karlcow  ?

I added a select to choose the view mode. I also added a function using localstorage to save this preference, if this feature is not interesting, tell me and I will delete it.

### CODE REVIEW

File: webcompat/static/js/lib/dashboard.js [hunk 1]
```diff
@@ -47,6 +47,27 @@ const filteringSort = issuesNumber => {
...
+  if (currentView === view) {
+    return;
+  }
+  /* set value */
+  Triages.dataset["view"] = view;
+  localStorage.setItem("DashboardTriageView", view);
+};
+
+const setView = (view = null) => {
+  if (null == view) {
```
Comment (miketaylr) [inline webcompat/static/js/lib/dashboard.js]: [yoda conditionals](https://en.wikipedia.org/wiki/Yoda_conditions) always seem weird to me. :p

File: webcompat/static/js/lib/dashboard.js [hunk 2]
```diff
@@ -47,6 +47,27 @@ const filteringSort = issuesNumber => {
   });
 };
 
+/* viewMode */
+const viewMode = () => {
+  const view = document.getElementById("js-Filter-view").value;
+  const Triages = document.getElementById("js-Triages");
```
Comment (miketaylr) [inline webcompat/static/js/lib/dashboard.js]: Most JS style recommends not using Uppercase except for classes/prototypes. Is Uppercase intentional here?

(it doesn't matter, just curious)

Comment (magsout) [reply webcompat/static/js/lib/dashboard.js]: Ooups no, it’s not.

File: webcompat/static/js/lib/dashboard.js (hunk 1 above)
Comment (magsout) [reply webcompat/static/js/lib/dashboard.js]: Ah ah 🙃

**Comment by miketaylr** [comment]
Ping @karlcow

File: webcompat/templates/dashboard/triage/needstriage-list.html [hunk 3]
```diff
@@ -101,54 +171,56 @@
...
+            {% for browser in issue['browsers'] %}
+              <span>{{ browser }} </span>
+            {% endfor %}
           </div>
           <div>
-            Updated:
-            <time
-              class="wc-Triage-date"
-              datetime="{{ issue['updated_at'] }}"
+            <a
```
[the remark this answers is not in the document]
Comment (magsout) [reply webcompat/templates/dashboard/triage/needstriage-list.html]: I remove the link `more information` and let the link around the title?

[the remark this answers is not in the document]
Comment (karlcow) [reply webcompat/templates/dashboard/triage/needstriage-list.html]: yup. :) we already have the link with the title.

**Comment by magsout** [comment]
@karlcow added your comment.

@miketaylr @karlcow shall we waiting for the proposition of karl ? or just merge now?

We can continue to iterate to something better.

**Comment by karlcow** [comment]
ship it

**Pull request merged**